spec/mailers/emailer_spec.rb

```diff
@@ -6,6 +6,19 @@
   include EmailSpec::Matchers
   fixtures :affiliates, :users, :features, :memberships
 
+  describe '#account_deactivation_warning' do
+    let(:user) { users(:not_active_76_days) }
+    let(:message) do
+      'automatically deactivate your account in the next 14 days due to inactivity.'
+    end
+
+    subject(:deactivate_email) { Emailer.account_deactivation_warning(user, 76.days.ago.to_date) }
```

Comment:
When describing a method, it is best to use a subject name consistent with the method being tested:
```ruby
subject(:account_deactivation_warning) do
  Emailer.account_deactivation_warning(user, 76.days.ago.to_date)
end
```